tests/acceptance/features/bootstrap/AppManagementContext.php

```diff
@@ -460,7 +453,7 @@ public function assertInstalledVersionOfAppIs(string $appId, string $version):vo
 		Assert::assertEquals(
 			$version,
 			\trim($cmdOutput),
-			"Expected: the installed version of ${appId} should be ${version} but got " . \trim($cmdOutput)
+			"Expected: the installed version of $appId should be $version but got " . \trim($cmdOutput)
```

Comment:
should it be `to be` or  `should be` is also correct?
```suggestion
			"Expected: the installed version of $appId to be $version but got " . \trim($cmdOutput)
```